rest/data/version.go

```diff
@@ -54,6 +54,24 @@ func (vc *DBVersionConnector) FindVersionById(versionId string) (*version.Versio
 	return v, nil
 }
 
+func (vc *DBVersionConnector) FindActivatedVersionsByProjectId(
+	projectId string,
+) ([]version.Version, error) {
+	versions, err := version.Find(version.ByProjectIdActivated(projectId))
+
+	if err != nil {
+		return nil, err
+	}
+
+	if versions == nil {
+		return nil, &rest.APIError{
```

Comment:
just return an error here - you can wrap it in a HTTP error in the route

Reply:
I think this should remain as is so that the correct status is returned into the route

Reply:
sgtm